scene/gui/button.cpp

```diff
@@ -457,6 +473,15 @@ Button::TextAlign Button::get_text_align() const {
 	return align;
 }
 
+void Button::set_icon_align(TextAlign p_align) {
+	icon_align = p_align;
+	update();
```

Comment:
```suggestion
	minimum_size_changed();
	update();
```

To update min. size when alignment is changed to/from `center`.

Reply:
Done.